lib/commands/moveToActive-8.lua

```diff
@@ -62,9 +62,12 @@ if jobId then
         -- put job into delayed queue
         rcall("ZADD", KEYS[7], timestamp * 0x1000 + bit.band(jobCounter, 0xfff), jobId)
         rcall("PUBLISH", KEYS[7], timestamp)
+        -- remove from active queue
+        rcall("LREM", KEYS[2], 1, jobId)
+        return
       end
-      -- remove from active queue
-      rcall("LREM", KEYS[2], 1, jobId)
+      -- move from active to wait
```

Comment:
use ```else``` for making intent more explicit.

Reply:
I am thinking actually if we put this code in an `if` statement and the move the code in the `if` statement right now outside, this would be more clear.